package.json

```diff
@@ -6,29 +6,33 @@
         "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
     },
     "main": "lib/index.js",
+    "types": "types/index.d.ts",
     "files": [
         "lib/",
-        "configs/"
+        "configs/",
+        "types/index.d.ts"
```

Comment:
I only exported this file as it solely references 'eslint' and none of the other files :eyes:

![image](https://github.com/eslint-community/eslint-plugin-n/assets/2230835/27b23d85-c38d-4610-86cc-2d2bbddc7a0b)

This is also why I only promoted `@types/eslint` to `dependencies`